### PR TITLE
added shimmer effect demo + modified parallax background color

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

File: lib/basic_effects/basic_effects_page.dart [hunk 1]
```diff
@@ -33,39 +35,44 @@ class _BasicEffectsPageState extends State<BasicEffectsPage>
...
+              }
 
+              return ListView.builder(
+                itemCount: snapshot.data.length,
+                itemBuilder: (context, index) => ShimmerListItem(
+                  index: index + 1,
+                ),
+              );
+            });
+
```
Comment (joykuotw) [inline lib/basic_effects/basic_effects_page.dart:70]: this should also be a view like case 0.

File: lib/basic_effects/basic_effects_page.dart [hunk 2]
```diff
@@ -93,12 +100,55 @@ class _BasicEffectsPageState extends State<BasicEffectsPage>
...
         children: [
           onSelectedWindow(0),
+          onSelectedWindow(1),
         ],
       ),
     );
   }
 }
 
+class parallaxView extends StatelessWidget {
```
Comment (joykuotw) [inline lib/basic_effects/basic_effects_page.dart:110]: The Class name should be UpperCamelCase.

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: lib/basic_effects/basic_effects_page.dart (hunk 1 above)
Comment (mysticeti) [reply lib/basic_effects/basic_effects_page.dart]: I have extracted the widget to ShimmerView in the next commit.

File: lib/basic_effects/basic_effects_page.dart (hunk 2 above)
Comment (mysticeti) [reply lib/basic_effects/basic_effects_page.dart]: Thank you for pointing that out. Changed the name to UpperCamelCase

**Review by joykuotw — APPROVED** [review]
I have removed the onSelectedWindow widget as it is not needed.

**Pull request merged**